source/FreeRTOS_IPv6_Sockets.c

```diff
@@ -143,7 +143,6 @@ size_t xRecv_Update_IPv6( const NetworkBufferDescriptor_t * pxNetworkBuffer,
                              ( const void * ) pxUDPPacketV6->xIPHeader.xSourceAddress.ucBytes,
                              ipSIZE_OF_IPv6_ADDRESS );
             pxSourceAddress->sin_family = ( uint8_t ) FREERTOS_AF_INET6;
-            pxSourceAddress->sin_address.ulIP_IPv4 = 0U;
```

Comment:
Thank you for this change, but I had it already staged in my #745

Reply:
Thanks for information. But we need to patch this first for protocol test.